src/databricks/labs/ucx/runtime.py

```diff
@@ -24,6 +24,104 @@
 logger = logging.getLogger(__name__)
 
 
+def _get_view_definition(hms_db) -> str:
+    return f"""CREATE
+OR REPLACE VIEW {hms_db}.vw_failure_summary AS WITH failuretab (failure, issue, object_type) AS (
+  SELECT
+    explode(from_json(failures, 'array<string>')) AS failure,
+    substring_index(failure, ":", 1) issue,
+    object_type
+  FROM
+    (
+      SELECT
+        failures,
+        "jobs" AS object_type
```

Comment:
Please also select object_id for every entity

Reply:
I think in that case we should have two views - one for summary as it is right now and the second one is more like a drill down with details such as object ids.

Reply:
@nfx